.github/workflows/build-deploy-tina-migration.yml

```diff
@@ -0,0 +1,35 @@
+name: Tina Migration - build and deploy
+
+on:
+  push:
+    branches:
+      - tina-migration
```

Comment:
I think the branch name should be `tina/main`, no?

Reply:
Done - i changed the target branch to tina/main